app/common/state/siteCache.js

```diff
@@ -97,3 +97,8 @@ const removeLocationSiteKey = (state, location, siteKey) => {
   }
 }
 module.exports.removeLocationSiteKey = removeLocationSiteKey
+
+const clearLocationSiteKeysCache = (state) => {
+  return state.set('locationSiteKeysCache', new Immutable.Map())
```

Comment:
I don't think it hurts anything, but new isn't needed here. immutable.js uses object.create, IIRC